package.json

```diff
@@ -20,7 +20,7 @@
     "test:watch": "karma start ./config/karma.conf.js",
     "lint": "eslint . --fix --max-warnings=0 --ext .ts",
     "prepare": "ts-patch install && npm run build",
-    "benchmark": "ts-node-script ./test/bench/**.ts"
+    "benchmark": "ts-node-script ./test/bench/document_test.ts"
```

Comment:
Is there a way to test files in `bench` folder without `glob`?

Reply:
`ts-node-script` needs an exact entry file path, so If we test files in bench folder without glob, I think we need to define the index file and load and run the entire test file

However, all the remaining tests are document tests (#306), so I didn't divide the files yet.

Reply:
In this PR we still have one test file, but what if we add an index.ts file and pre-create the structure?

Reply:
I think it's good. I'll try it out. Thank you for your suggestion!